### PR TITLE
Add tango-database recipe

### PR DESCRIPTION
**Pull request by beenje**
Checklist
- [X] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [X] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [X] Source is from official source.
- [X] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [X] If static libraries are linked in, the license of the static library is packaged.
- [X] Build number is 0.
- [X] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [X] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [X] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by t-b** [comment]
@beenje I'd say this looks good.

> The build is done with mysql 8 (version available on conda-forge). Is it OK?

Yes let's give it a shot.

> Does one of you want to be added as maintainer?

I'd prefer to pass that offer. And concentrate on cppTango/TSD.

**Comment by bourtemb** [comment]
@beenje , you can add me as maintainer too for the moment.

**Review by bourtemb — APPROVED** [review]
Many thanks @beenje .
This looks good to me.
Feel free to add me as maintainer with you.

**Comment by beenje** [comment]
@conda-forge/help-c-cpp this PR is ready for review!

File: recipes/tango-database/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    - {{ compiler('cxx') }}
+    - libtool
+    - pkg-config
+  host:
+    - cppzmq
+    - cpptango
+    - mysql-devel
+  run:
+    # cpptango set in run_exports
+    - {{ pin_compatible('omniorb', min_pin='x.x', max_pin='x.x') }}
```
Comment (chrisburr) [inline recipes/tango-database/meta.yaml:29]: `omniorb` should probably be a `host` dependency

Comment (beenje) [reply recipes/tango-database/meta.yaml]: I removed the explicit omniorb run dependency. It's already installed as cpptango dependency.

Comment (chrisburr) [reply recipes/tango-database/meta.yaml]: You should add it if you're linking to it (which you are). Everything you link to should be explicitly listed in `host` so the bots know how to perform ABI migrations without building the entire world every time.

Comment (beenje) [reply recipes/tango-database/meta.yaml]: Good point. Thanks. I added it back.

**Comment by beenje** [comment]
@chrisburr, does the recipe looks ok to you now?

**Comment by mguijarr** [comment]
@beenje @bourtemb just to say I tested this package at ESRF on a new Ubuntu computer : it works like a charm. Thanks a lot guys !

Little remark: mysql is not a runtime dependency ; it has to be installed separately. Maybe it's fine.

**Comment by beenje** [comment]
> @beenje @bourtemb just to say I tested this package at ESRF on a new Ubuntu computer : it works like a charm. Thanks a lot guys !
> 
> Little remark: mysql is not a runtime dependency ; it has to be installed separately. Maybe it's fine.

Thanks for the feedback!

Regarding mysql dependency, that's correct. I didn't want to force installing it. Mysql could be installed on another machine or it could be installed outside conda (with the OS package manager). If you want to have it in the same conda env, it's easy to specify it.

**Comment by bourtemb** [comment]
> Mysql could be installed on another machine or it could be installed outside conda (with the OS package manager). If you want to have it in the same conda env, it's easy to specify it.

Still there should be a runtime dependency to the mysql client library? Isn't it?

**Comment by beenje** [comment]
> > Mysql could be installed on another machine or it could be installed outside conda (with the OS package manager). If you want to have it in the same conda env, it's easy to specify it.
> 
> Still there should be a runtime dependency to the mysql client library? Isn't it?

Yes. The `mysql-devel` dependency in the host section automatically adds a run dependency on `mysql-libs` due to the run_exports: https://github.com/conda-forge/mysql-feedstock/blob/27e602dde2e96f6eccbca4884a72085d2873333f/recipe/meta.yaml#L126

**Comment by bourtemb** [comment]
Yes, this is what I just saw! Perfect! Thanks @beenje for this recipe.
Thanks @chrisburr for the review and thanks @mguijarr for the feedback!

**Pull request merged**